### PR TITLE
[TASK] Align with new TYPO3 documentation standards

### PR DESCRIPTION
**Pull request by alexander-nitsche**
Relates: https://github.com/TYPO3-Documentation/T3DocTeam/issues/182

### CODE REVIEW

**Comment by alexander-nitsche** [comment]
What is missing: Registering this documentation for rendering and publishing at docs.typo3.org with the [webhook](https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/WritingDocForExtension/Webhook.html#github).

**Comment by alexander-nitsche** [comment]
How it looks:
Screen Shot 2022-04-03 at 18 20 16

**Comment by alexander-nitsche** [comment]
Added two new commits regarding badges in README and additional sources (TER, repository and docs.typo3.org) in composer.json.

In addition, the TER homepage is missing the "Packagist" buttons, which seem to be caused by the empty "Composer name of extension" fields in the TER extension edit form.

Relates: TYPO3-Documentation/T3DocTeam#185

File: Documentation/Index.rst [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+=======
+
+The extension is published under GPL v2+.
+
+Authors
+=======
+
+A lot of contributors have been working on this area while this functionality
+was part of the TYPO3 Core. This package is now maintained by a loose group of
+TYPO3 enthusiasts inside the TYPO3 Community. Feel free to contact `Benni Mack`_
```
Comment (ohader) [inline Documentation/Index.rst]: Please avoid pointing to Benni here, refer to the issue tracker instead...
(seems that basically all legacy extensions had that, I wonder why...)

**Review by ohader — CHANGES_REQUESTED** [review]
Avoid author pointing...

File: Documentation/Index.rst (hunk 1 above)
Comment (alexander-nitsche) [reply Documentation/Index.rst]: Done.

**Review by ohader — APPROVED** [review]

**Comment by ohader** [comment]
TER foo pending due to wrong composer name `friendsoftypo3/func` of uploader TER archive

**Comment by alexander-nitsche** [comment]
See follow-up https://github.com/FriendsOfTYPO3/func/issues/7.

**Pull request merged**